### PR TITLE
add docstring for `EnerFitting`

### PR DESCRIPTION
**Pull request by njzjz**
Screenshot from 2021-08-24 18-34-05

### CODE REVIEW

File: deepmd/fit/ener.py [hunk 1]
```diff
@@ -14,17 +14,46 @@
...
-    """Fitting the energy of the system. The force and the virial can also be trained.
+    r"""Fitting the energy of the system. The force and the virial can also be trained.
+
+    The potential energy :math:`E` is a fitting network function of the descriptor :math:`\mathcal{D}`:
+
+    .. math::
+        E(\mathcal{D}) = \mathcal{L}^{(n)} \circ \mathcal{L}^{(n-1)}
+        \circ \cdots \circ \mathcal{L}^{(1)} \circ \mathcal{L}^{(0)}
+
+    The first :math:`n` layers :math:`\mathcal{L}^{(0)}, \cdots, \mathcal{L}^{(n-1)}` is given by
```
Comment (amcadmus) [inline deepmd/fit/ener.py:25]:
```suggestion
    The first :math:`n` hidden layers :math:`\mathcal{L}^{(0)}, \cdots, \mathcal{L}^{(n-1)}` are given by
```

File: deepmd/fit/ener.py [hunk 2]
```diff
@@ -14,17 +14,46 @@
...
+        \mathbf{y}=\mathcal{L}(\mathbf{x};\mathbf{w},\mathbf{b})=
+            \boldsymbol{\phi}(\mathbf{x}^T\mathbf{w}+\mathbf{b})
+
+    where :math:`\mathbf{x} \in \mathbb{R}^{N_1}`$` is the input vector and :math:`\mathbf{y} \in \mathbb{R}^{N_2}`
+    is the output vector. :math:`\mathbf{w} \in \mathbb{R}^{N_1 \times N_2}` and
+    :math:`\mathbf{b} \in \mathbb{R}^{N_2}`$` are weights and biases, respectively,
+    both of which are trainable if `trainable[i]` is `True`. :math:`\boldsymbol{\phi}`
+    is the activation function.
+
+    The last layer :math:`\mathcal{L}^{(n)}` is given by
```
Comment (amcadmus) [inline deepmd/fit/ener.py:37]:
```suggestion
    The output layer :math:`\mathcal{L}^{(n)}` is given by
```

**Review by amcadmus — COMMENTED** [review]
Do you know how a document contributor can quickly preview the document with math?

**Comment by njzjz** [comment]
> Do you know how a document contributor can quickly preview the document with math?

Install with edit mode `pip install -e .` and regenerate using `make html`... I don't have other ideas.

**Review by amcadmus — APPROVED** [review]

**Pull request merged**